curriculum/challenges/english/20-upcoming-python/learn-string-manipulation-by-building-a-cipher/6555e3f387381a19d5e00333.md

```diff
@@ -0,0 +1,69 @@
+---
+id: 6555e3f387381a19d5e00333
+title: Step 79
+challengeType: 20
+dashedName: step-79
+---
+
+# --description--
+
+Do the same with the other two `print` calls.
```

Comment:
In general, I would avoid lessons like this. If a Camper leaves at this step, and comes back in 5, 10, or 1000 minutes time, they might have forgotten what "the same" is.